.circleci/config.yml

```diff
@@ -0,0 +1,106 @@
+version: 2.1
+
+jobs:
+  bundle-client-dependencies:
+    working_directory: ~/loopback-to-do/client
+    docker:
+      - image: circleci/node:11.6
+    steps:
+      - checkout:
+          path: ~/loopback-to-do
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: Client setup
+          command: yarn
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - node_modules
+      - persist_to_workspace:
+          root: .
+          paths:
+            - node_modules
+  bundle-server-dependencies:
+    working_directory: ~/loopback-to-do/server
+    docker:
+      - image: circleci/node:11.6
+    steps:
+      - checkout:
+          path: ~/loopback-to-do
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: Server setup
+          command: yarn
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - node_modules
+      - persist_to_workspace:
+          root: .
+          paths:
+            - node_modules
+  lint-client:
+    working_directory: ~/loopback-to-do/client
+    docker:
+      - image: circleci/node:11.6
+    steps:
+      - checkout:
+          path: ~/loopback-to-do
+      - attach_workspace:
+          at: ~/loopback-to-do/client
+      - run:
+          name: Lint client
+          command: ./node_modules/.bin/eslint src
```

Comment:
@nikitasakov can't we use `yarn lint` for that?

Reply:
When I've tried - it is didn't know this command, I think it is because eslint wasn't installed globally on circleci server.